File: test/fixtures/csharp-slop/src/OrderService.cs
using System;

namespace Acme
{
    public class OrderService
    {
        public void Submit(string id)
        {
            try
            {
                Persist(id);
            }
            catch (Exception e)
            {
            }
        }

        public void Export()
        {
            throw new NotImplementedException();
        }

        private void Persist(string id)
        {
            if (string.IsNullOrEmpty(id))
            {
                throw new ArgumentException("id required");
            }
            Console.WriteLine(id);
        }
    }
}

File: test/fixtures/java-slop/src/main/java/OrderService.java
public class OrderService {

    public void submitOrder(String id) {
        try {
            persist(id);
        } catch (Exception e) {
        }
    }

    public void auditOrder(String id) {
        try {
            persist(id);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

    public void exportOrders() {
        throw new UnsupportedOperationException("not implemented yet");
    }

    private void persist(String id) {
        if (id == null || id.isEmpty()) {
            throw new IllegalArgumentException("id required");
        }
        System.out.println(id);
    }
}
